lib/ansible/modules/windows/win_firewall_rule.py

```diff
@@ -34,6 +34,12 @@
       - The rule's display name.
     type: str
     required: yes
+  group:
+    description:
+      - The group name for the rule.
+    version_added: '2.9'
+    type: str
+    required: yes
```

Comment:
Why did you mark it as required?

Reply:
ohho,forgot to remove it after copy paste. I've removed it,